packages/keymaps/src/browser/keymaps-frontend-module.ts

```diff
@@ -21,9 +23,14 @@ export default new ContainerModule(bind => {
         ).inSingletonScope();
     }
 
+    bind(UserStorageWatcher).toSelf().inSingletonScope();
+
     bind(FrontendApplicationContribution).to(KeymapsService).inSingletonScope();
 
-    bind(KeymapsServer).toDynamicValue(ctx =>
-        ctx.container.get(WebSocketConnectionProvider).createProxy(keybindingsPath)
+    bind(StorageServer).toDynamicValue(ctx =>
```

Comment:
`StorageServer` is too generic name, better `UserStorageServer` or `IUserStorageServer` if the former is alread taken